helm/plain-node/templates/plain-node.yaml

```diff
@@ -1,39 +1,39 @@
 apiVersion: apps/v1
```

Comment:
why aren't plain nodes assimilated into generic nodes?  a plain node is just a node that doesn't have a key and so it participates in the network but it just doesn't stake anything.  i guess a better name could be "non staking node"